src/mechaclient.py

```diff
@@ -49,16 +49,24 @@ def __init__(self, *args, **kwargs):
         self._fact_manager = None  # Instantiate Global Fact Manager
         self._rat_cache = None  # TODO: replace with ratcache once it exists
         self._rat_board = None  # Instantiate Rat Board
+        self._config = mecha_config if mecha_config else {}
         super().__init__(*args, **kwargs)
 
+    @property
```

Comment:
[Question]  Do we need a public property here?  I would think config should remain private across the module, since each module has its own config object now.

Reply:
No we do not, each module gets its own private property in this regard, and shouldn't be getting it from the mechaclient. will fix.